### PR TITLE
yaml-cpp 0.7.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
trafficserver does not build :(

**Comment by chenrui333** [comment]
relates to #83326

**Comment by cho-m** [comment]
Patch is failing everywhere, but CI passes because only Catalina & Mojave bottle exist.

Need to backport patch as some modified lines don't exist, e.g. https://github.com/apache/trafficserver/blob/9.1.0/mgmt/Makefile.am#L36-L37

File:
```
	-I$(abs_top_srcdir)/proxy/hdrs \
	$(TS_INCLUDES)
```

Patched part
```diff
 	-I$(abs_top_srcdir)/proxy/hdrs \
-	-I$(abs_top_srcdir)/lib/yamlcpp/include \
+	@YAMLCPP_INCLUDES@ \
 	$(TS_INCLUDES)
```

File: Formula/trafficserver.rb [hunk 1]
```diff
@@ -48,7 +70,7 @@ def install
       --enable-experimental-plugins
     ]
 
-    system "autoreconf", "-fvi" if build.head?
+    system "autoreconf", "-fvi" if build.head? || OS.mac?
```
Comment (cho-m) [inline Formula/trafficserver.rb:73]: This worked for me locally on Monterey.
Could also just `autoreconf` everywhere including Linux (as only Linux stable isn't running it right now. Not sure if it builds yet).

**Review by iMichka — APPROVED** [review]
Let's ship this, looks good.

**Pull request closed without merging**